docs/src/experimental/intro.md

```diff
@@ -0,0 +1,58 @@
+# Adding new projects to experimental
+
+## Purpose
+The folder `experimental` is for code that is candidate for being added to
+Oscar. In particular, this includes the following cases:
+- Code from an external package that should be moved to Oscar
+- Implementing a new feature from scratch
+- In general: code whose API has not stabilized yet
+
+The code in `experimental` is supposed to be mathematically correct,
+`experimental` is supposed to be a staging area for new features whose
+interface is still to be stabilized. Also code is allowed to reside in
+`experimental` while it is brought up to Oscar standard.
+
+## Structure
+To look at a sample for the structure for a new project in `experimental` have
+a look at `experimental/PlaneCurve`. The general structure is
+```
+experimental\
+   PACKAGE_NAME\
```

Comment:
Hmm, maybe this?
```suggestion
experimental/
   PACKAGE_NAME/
```

Reply:
Thanks. But I wanted to ask you anyway: How did you get it to look so nice in #2146 ?

Reply:
I used the `tree` command line tool (see <http://mama.indstate.edu/users/ice/tree/> but I installed it via Homebrew, it is also available for most Linux distros)